PWGHF/TableProducer/HFTrackIndexSkimsCreator.cxx

```diff
@@ -421,15 +423,25 @@ struct HfTagSelTracks {
 
       iDebugCut = 4;
       if (doCutQuality.value && (debug || statusProng > 0)) { // FIXME to make a more complete selection e.g track.flags() & o2::aod::track::TPCrefit && track.flags() & o2::aod::track::GoldenChi2 &&
-        UChar_t clustermap = track.itsClusterMap();
-        if (!(track.tpcNClsFound() >= tpcNClsFound.value && // is this the number of TPC clusters? It should not be used
-              track.flags() & o2::aod::track::ITSrefit &&
-              (TESTBIT(clustermap, 0) || TESTBIT(clustermap, 1)))) {
+        bool hasGoodQuality = true;
+        if (useIsGlobalTrack) {
+          if (track.isGlobalTrack() != (uint8_t) true) {
```

Comment:
> `!= (uint8_t) true`

Why is this weird type conversion needed?
Does `isGlobalTrack` not return `bool`?

Reply:
The column type in Common/DataModel/TrackSelectionTables.h is actually uint8_t, I think that removing the explicit conversion would still work fine, but I left that conversion for full consistency (it's also how the check is done in the qa task). nyway, if you prefer to remove the explicit casting, I can do it